app/AppKernel.php

```diff
@@ -26,6 +26,7 @@ public function registerBundles()
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
+            new Symfony\Bundle\WebServerBundle\WebServerBundle(),
```

Comment:
You should register the server bundle only for dev environment. You should move it after line 41.